int-test

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+npm run build
+docker-compose down
```

Comment:
It'a nice to have but ideally it would be nice to be able to run the int tests locally without interfering with a locally running build.

Reply:
ok